### ISSUE TITLE
[TASK] Question: iOS/Android difference on Privacy level slider use

### ISSUE THREAD

**Issue by acovher2** [rokwire/illinois-app#5745]
### Description
Please see both videos for Android and iOS Privacy slider use. 

https://github.com/user-attachments/assets/d9848a18-356c-4081-9e51-1c5cc17ab723

https://github.com/user-attachments/assets/9c043393-fcfe-4b4f-b224-9c7f4c6381dd

Android - The slider instructions work. 
iOS - The slider instructions work only after double tapping on the slider, then the user can slide up or down respectively. 

### Question: 
Is there a way to circumvent the need to double tap on slider for iOS?
or, can we include the double tap in the assistive technology labeling to better instruct users_. If I had not figured the double tap out, I was about to open a bug where I was unable to use the slider on iOS only._
